static/styles/portico.scss

```diff
@@ -1233,8 +1243,6 @@ input#terminal:checked ~ #tab-terminal {
 
 #integration-list-link {
     position: absolute;
-    top: 200px;
```

Comment:
Okay. So, the thing is that, this part of the CSS is still there. I just have shifted it to the`@media (min-width: 768px)` which basically means that this all values will apply to screen above 768px. 

I've done this for obvious reasons as mentioned in the other comment on this PR.

Reply:
Okay. So, the thing is that, this part of the CSS is still there. I just have shifted it to the`@media (min-width: 768px)` which basically means that this all values will apply to screen above 768px. 

I've done this for obvious reasons as mentioned in the other comment on this PR.

Reply:
ohk @sameerchoubey,  I used my master branch for the testing and seems like the error was rising because of the browser cache, idk for sure but rechecked after re-running the browser and reloading with `ctrl + F5` made it work.
P.S. : everything works fine :+1: